weblate_web/templates/about.html

```diff
@@ -44,6 +44,36 @@ <h2>{% trans "Weblate grows and the future is open to new opportunities." %}</h2
                 </div>
             </div>
 
+            <h2 class="section-title crypto">{% trans "Your Weblate Team" %}</h2>
+            <div class="weblate-team">
+                <div class="weblate-team-content">
+                   <img src="https://hosted.weblate.org/avatar/80/nijel.png" alt="Michal Čihař" />
+                     <div class="content-p">
+                       <p>{% trans "Michal Čihař" %}</p>
+                       <br>
+                       <t>{% trans "Master of the code & head of custom or large scale solutions." %}</t>
+                    </div>
+                </div>
+                <div class="weblate-team-content">
+                    <img src="https://hosted.weblate.org/avatar/80/orangesunny.png" alt="Benjamin Alan Jamie" />
+                    <div class="content-p">
+                        <p>{% trans "Benjamin Alan Jamie" %}</p>
+                        <br>
+                        <t>{% trans "Community manager & support guru helping with smooth workflows." %}</t>
```

Comment:
Let’s use:

*Community manager, support guru, helping users with smooth workflows; and the one to talk sales.*

Reply:
Done!